MapboxMobileEvents/MMEMetricsManager.m

```diff
@@ -168,12 +233,12 @@ - (NSDictionary *)attributes {
     attributes[MMEEventEventCountMax] = @(self.metrics.eventCountMax);
     attributes[MMEEventAppWakeups] = @(self.metrics.appWakeups);
     attributes[MMEEventRequests] = @(self.metrics.requests);
-    attributes[MMEEventDeviceTimeDrift] = @(self.metrics.deviceTimeDrift);
+    attributes[MMEEventDeviceTimeDrift] = @(MMEDate.recordedTimeOffsetFromServer);
     if (self.metrics.deviceLat != 0 && self.metrics.deviceLon != 0) {
         attributes[MMEEventDeviceLat] = @(self.metrics.deviceLat);
         attributes[MMEEventDeviceLon] = @(self.metrics.deviceLon);
     }
-    
+    attributes[MMEEventKeyVendorID] = [MMEEventsManager sharedManager].commonEventData.vendorId;
```

Comment:
This should be removed. I added when I thought it was a different metric -- it's not needed and not accepted by our schema. (equates to `userId`)

Reply:
Must have picked it up in a rebase conflict, removing